docs/kgcPy2.py

```diff
@@ -0,0 +1,20 @@
+'''Vectorized version of functions in the kgcPy package'''
```

Comment:
```suggestion
'''Accelerated version of functions in the kgcPy package'''
```

Not "vectorized" in the usual sense; the function still only processes one location at a time.